addons/kommander/1.x/kommander-11.yaml

```
@@ -41,7 +41,7 @@ spec:
   chartReference:
     chart: kommander
     repo: https://mesosphere.github.io/charts/stable
-    version: 0.4.8
+    version: 0.4.10
```

As per https://github.com/mesosphere/charts/pull/447

```suggestion
    version: 0.4.12
```